tests/metadata-service-tests/test/k8stests/k8sresource_test.go

```diff
@@ -54,6 +55,9 @@ func TestK8sResources(t *testing.T) {
 
 		expectedLabels := map[string]string{"re": dummyRE.Name, "serviceId": serviceId}
 
+		// wait for changes to propagate in the CR
+		time.Sleep(10 * time.Second)
```

Comment:
I suggest to export `10 * time.Second` to some const like `waitForChangesPropagationInCR`. No comments would be needed and the value would be changeable in easier way.